docs/dev/dev-proxy/how-to/uninstall.md

```diff
@@ -9,37 +9,50 @@ zone_pivot_groups: client-operating-system
 
 # Uninstall
 
-If you want to remove Dev Proxy from your machine, follow these steps.
+If you want to remove `Dev Proxy` from your machine, follow these steps.
 
 ::: zone pivot="client-operating-system-windows"
 
-1. Uninstall Dev Proxy by running in the command prompt: `winget uninstall Microsoft.DevProxy`
-1. Remove the `Dev Proxy` certificate from your machine.
-    1. Open `Start Menu`.
-    1. In the search box, Enter `Manage user certificates`.
-    1. To open the `Certificates` dialog box, select the result in the list.
-    1. In the tree view, expand the `Personal` folder and select the `Certificates` child folder.
-    1. Remove the `Dev Proxy CA` certificate by selecting and pressing the <kbd>Delete</kbd> key on your keyboard, or right select and in the menu select `Delete`.
-    1. To confirm the deletion, Select `Yes`.
+1. Uninstall `Dev Proxy` by running in the command prompt: `winget uninstall Microsoft.DevProxy`
+    > [!NOTE]
+    > The uninstaller removes `Dev Proxy` application and its certificate.
+1. Optional. Manually remove the `Dev Proxy` certificate from your machine.
```

Comment:
If the installer automatically removes the certificate, when would users go through this step?

Reply:
I suggest we keep all this information in one place like we do now. Otherwise, we'll keep sending people around our docs for a single job: removing Dev Proxy and its resources from their machine. What we also could do, is separate the certificate removal instructions from uninstalling, and move it to a separate section further in the page with additional explainer like "In case you chose not to remove the certificate while uninstalling Dev Proxy, follow these steps to remove it manually:" and then use tabs to switch between Windows and macOS like we do for uninstallation.